app/helpers/application_helper.rb

```diff
@@ -148,7 +148,7 @@ def ssn(number)
     if can_view_full_ssn?
       # pad with leading 0s if we don't have enough characters
       number = number.to_s.rjust(9, '0') if number.present?
-      content_tag :span, number.to_s.gsub(HudUtility2024::SSN_RGX, '\1-\2-\3')
+      content_tag :span, number.to_s.gsub(HudHelper.util::SSN_RGX, '\1-\2-\3')
```

Comment:
this can be done later but this reference assumes that HudHelper.util returns a class. There aren't that many places where we do this; might be worth replacing `HudHelper.util::X` - maybe just add methods directly to HudHelper
```suggestion
      content_tag :span, HudHelper.format_ssn(number)
```

Reply:
What do you think about `HudHelper.constants('SSN_RGX')` That could take an optional version `HudHelper.constants('SSN_RGX', version: '2024')` in case we end up having different constants per year.  For now the `constants` method could just call `util::SSN_RGX`, but we could upgrade it in the future as necessary.

Reply:
> What do you think about HudHelper.constants('SSN_RGX')

I guess exposing constants seems okay as a bridge to something else. For SSNs in the long term I think we planned to remove the `ssn` and `masked_ssn` helpers and instead wrap data in `GrdaWarehouse::PiiProvider` - that would centralize formatting and enforce PII visibility. 

The other place we reference Util constants appears to be for ranges (`HudHelper.util::SITUATION_HOMELESS_RANGE`). 

> That could take an optional version HudHelper.constants('SSN_RGX', version: '2024') in case we end up having different constants per year. 

I guess I'd favor using simple method wrappers rather than add another access path. Methods seem more direct and consistent how the rest of the behavior is works ( so `HudHelper.util.ssn_rgx`)

```ruby
#  module HudUtility2026
#    include ::Concerns::HudUtilityCommonBehaviors
 
module Concerns
  module HudUtilityCommonBehaviors
    extend ActiveSupport::Concern

    class_methods do
      # expose internal constants
      def ssn_rgx = self::SSN_RGX
      def situation_other_range = self::SITUATION_OTHER_RANGE
      def situation_homeless_range = self::SITUATION_HOMELESS_RANGE
      def situation_institutional_range = self::SITUATION_INSTITUTIONAL_RANGE
      def situation_temporary_range = self::SITUATION_TEMPORARY_RANGE
      def situation_permanent_range = self::SITUATION_PERMANENT_RANGE
    end
  end
end
```